frontend/app/controllers/spree/store_controller.rb

```diff
@@ -13,14 +13,19 @@ def unauthorized
       render 'spree/shared/unauthorized', layout: Spree::Config[:layout], status: 401
     end
 
+    def account_link
+      render partial: 'spree/shared/link_to_account'
+      fresh_when(try_spree_current_user)
```

Comment:
Shouldn't we add `public: true` option here?

Reply:
@piotrleniec-spark definitely not, this is per user and should be private